Dockerfile

```diff
@@ -27,6 +27,7 @@ RUN set -ex; \
         python3-devel \
         libpq \
         libpq-devel \
+        git \
```

Comment:
According to our last discussion `git` is not going to be used from within container. Is this dependency still required?

Reply:
Yes. The script throws an error if git isn't available that's different from the one if git is run outside of a repo, and I don't think it's a good idea to ignore that error.